.github/workflows/publish.yml

```diff
@@ -1,13 +1,21 @@
 name: 'Package Publish'
 on:
-  push:
-    branches: [master]
+  release:
+    types: [ created ]
 
 jobs:
   publish:
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
     steps:
       - uses: actions/checkout@v2
-      # TODO
-#      - - name: NPM Publish
-#          uses: JS-DevTools/npm-publish@v1
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14.x'
+          registry-url: 'https://npm.pkg.github.com'
+      - run: npm install
```

Comment:
```suggestion
      - run: npm ci
```

Reply:
If `yarn.lock` work there I can do this, otherwise we have to add the package-lock.json

Reply:
This should then be `yarn` and not `npm install`, though?

Reply:
Oh shit, yes absolutely! `yarn install --frozen-lockfile` should be the equivalent, then, I believe.